mpf/platforms/virtual.py

```diff
@@ -311,21 +315,23 @@ class VirtualSegmentDisplay(SegmentDisplayPlatformInterface):
 
     """Virtual segment display."""
 
-    __slots__ = ["text", "flashing", "colors"]
+    __slots__ = ["text", "flashing", "platform_options", "colors", "machine", "post_update_events"]
 
-    def __init__(self, number) -> None:
+    def __init__(self, number, platform_options, machine) -> None:
```

Comment:
Guess we should remove this again right?

Reply:
Yes. I missed that one.